aikau/src/main/resources/alfresco/renderers/_XhrActionsMixin.js

```diff
@@ -46,6 +46,29 @@ define(["dojo/_base/declare",
        * @since 1.0.46
        */
       i18nRequirements: [{i18nFile: "./i18n/_XhrActionsMixin.properties"}],
+      
+      /**
+       * The lookup key into current item to retrieve the short name of the site in which the current item is
+       * located. This may be relevant to correctly obtain all item actions if evaluation of these actions is
+       * dependant on the site scope.
+       * 
+       * @instance
+       * @type {string}
+       * @default null
```

Comment:
When the initial assignment is `null`, a String, number or a boolean it is not necessary to set it on the `default` annotation... JSDoc automatically figures that out.